src/helpers/randomizeCard/index.js

```diff
@@ -0,0 +1,736 @@
+eff('toad',new Effect('spawn a Toad with {value} strength{value2} {space}',[3,3,1,1,1],1))
+eff('toad',new Effect('drain {value} strength from {target} enemy {target2}{forEach}',[1,0,1,1,1],0.5))
+
+eff('rodent',new Effect('push {targetPush}',[3,3,3,3,3],0.5))
+eff('rodent',new Effect('pull {targetPull}',[3,0,3,3,3],0.5))
+eff('rodent',new Effect('push {targetPush} and deal {value} damage to it{forEach}',[3,3,3,3,3],0.75))
+eff('rodent',new Effect('pull {targetPull} and deal {value} damage to it{forEach}',[3,0,3,3,3],0.75))
+eff('rodent',new Effect('deal {value} damage to {target} enemy {target2}{forEach}',[3,3,3,3,3],0.75))
+eff('rodent',new Effect('deal {valueHigh} damage to {target} enemy {target2}',[3,3,3,3,3],0.75))
+eff('rodent',new Effect('deal {valueHigh} damage spread randomly among {targetSpread}',[3,3,3,3,3],0.75))
+
+eff('construct',new Effect('give {value} strength to {target} friendly {target2}{forEach}',[3,3,3,3,3],1))
+eff('construct',new Effect('give {value} strength to {target} friendly {target2} and itself{forEach}',[3,0,3,3,3],1.5))
+eff('construct',new Effect('vitalize {target} friendly {target2}',[1,1,1,1,1],0.5))
+eff('construct',new Effect('vitalize {target} friendly {target2} and itself',[3,0,0,0,0],1))
+eff('construct',new Effect('spawn a Construct with {value} strength {space}',[0,3,3,3,3],1))
+
+eff('frostling',new Effect('freeze itself',[1,0,0,1,0],-0.75))
+eff('frostling',new Effect('freeze {target} friendly {target2}',[1,1,1,1,1],-0.75))
+eff('frostling',new Effect('freeze {target} enemy {target2}',[3,3,3,3,3],1))
+eff('frostling',new Effect('freeze {target} enemy {target2} and deal {value} damage to it{forEach}',[3,3,3,3,3],1))
+eff('frostling',new Effect('deal {value} damage to {target} enemy {target2}{forEach}',[3,3,3,3,3],0.75))
+eff('frostling',new Effect('gain {value} mana{forEach}',[3,0,3,0,3],1))
+
+eff('dwarf',new Effect('gain {value} strength{forEach}',[0,0,3,3,3],0.75))
+eff('dwarf',new Effect('give {value} strength to {target} friendly {target2}{forEach}',[3,3,3,3,3],0.75))
+eff('dwarf',new Effect('lose {value} strength{forEach}',[0,0,3,3,3],-0.5))
+
+eff('undead',new Effect('spawn an Undead with {value} strength {space}',[3,3,3,3,3],1))
+eff('undead',new Effect('deal {value} damage to the enemy base',[3,3,3,3,3],1))
+eff('undead',new Effect('command {target} friendly {target2} forward',[3,3,3,3,3],1))
+eff('undead',new Effect('give {value} strength to {target} friendly {target2}{forEach} and command it forward',[3,3,3,3,3],1))
+
+eff('satyr',new Effect('spawn a Satyr with {value} strength {space}',[3,0,3,3,3],1))
+eff('satyr',new Effect('give {value} strength to {target} friendly {target2}{forEach}',[3,3,3,3,3],0.75))
+eff('satyr',new Effect('vitalize {target} friendly {target2}',[3,3,3,3,3],0.75))
+eff('satyr',new Effect('deal {value} damage to your base',[3,3,3,3,3],-1))
+
+var slots = [
+	new Slot('target','a random bordering','unit',1),
+	new Slot('target','a random surrounding','unit',1.25),
```

Comment:
One thing we’ll need to do is automated highlighting as well. For words like bordering and surrounding. We can check that later though!